portfolio.html

```diff
@@ -8,6 +8,284 @@
+            <li class="menu menu-items3"><a href="#"></a>Contact</li>
+        </ul>
+    </header>
+
+    <!-- Desktop version headline -->
+    <section class="desktop-headline">
+        <div class="desktop-headline-first-text">
+            <h2 class="desktop-headline-heading">Hey there. I’m Norbi<br><span>I’m a software developer</span></h2>
+        </div>
+
+        <div class="desktop-headline-supporting-text">
+            <p class="desktop-headline-secondary-text">
+                I can help you build a product , feature or website Look through some of my work and <br> experience! If you like what you see and have a project you need coded, <br>
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <ul class="desktop-social-media-icons">
+            <li><a href="https://github.com/norbiboy1"><img src="./images/NormalButton2/Tertiary/Icons/github.png" class="github" alt="Github icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/siko-norbert/"><img src="./images/NormalButton2/Tertiary/Icons/linkedin.png" class="linkedin" alt="Linkedin icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/angelis.png" class="angelis" alt="Angelis icon"></a></li>
+            <li><a href="https://twitter.com/NorbertSiko"><img src="./images/NormalButton2/Tertiary/Icons/twitter.png" class="twitter" alt="Twitter icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/Vector.png" class="vector" alt="Vector icon"></a></li>
+        </ul>
+
+        <div class="desktop-normal-button">
+            <button type="button" class="btn2">
+                <img src="./images/NormalButton2/Secondary/Disabled.png" class="desktop-disabled-button" alt="Arrow down icon">
+            </button>
+        </div>
+    </section>
+
+    <!-- Desktop version work section -->
+    <section class="desktop-work">
+        <div class="desktop-recent-works">
+            <p class="desktop-primary-text">
+                My Recent Works
+            </p>
+        </div>
+
+        <div class="desktop-card-works">
+            <div class="desktop-card-work-row-1">
+                <div class="desktop-card-1">
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="desktop-button-frame">
+                        <button type="button" class="desktop-btn-green">See Project</button>
```

Comment:
- [ ] Kindly implement the **hover** and after **pressing** and **exact** color of all the buttons on your page, you can check them by clicking on the figma on the button
![image](https://user-images.githubusercontent.com/62705947/186937115-09df4f23-bf45-408a-b304-e284a7aeadea.png)